CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Full documentation for rocBLAS is available at [rocblas.readthedocs.io](https://rocblas.readthedocs.io/en/latest/).
 
+## For rocBLAS 3.1.0 in ROCm 5.7.0
+
+### Changed
+- Added trmv example to illustrate how different data types for handling complex numbers can be used
```

Comment:
Could it be worded as:
Added complex version of trmv (ctrmv) example to illustrate the absolute and relative error checking when initialized with uniform real numbers.

Reply:
Well I can add that but then will need to update more.  I'll can update this PR as there is no rush on this as waiting on an updated build it looks like.